src/pudl/transform/ferc1.py

```diff
@@ -355,6 +362,71 @@ def align_row_numbers_dbf(
     return df
 
 
+class SelectDbfRowsFromCategory(TransformParams):
+    """Parameters for :func:`select_dbf_rows_from_category`."""
+
+    column_name: str | None = None
+    select_based_on_xbrl_category: bool = False
```

Comment:
Maybe `select_by_xbrl_category` and also use `_by_` in the function name below, rather than having "from" and "based on" talking about the same thing, and colliding with the `SELECT * FROM table` nomenclature of SQL.

Reply:
wow zane suggesting a **shorter** name! love this lol. this is a great suggestion